File: 프로그래머스/lv1/131705. 삼총사/삼총사.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(vector<int> number) {
    int answer = 0;

    for(int i=0; i<number.size(); i++) {
        for(int j=i+1; j<number.size(); j++) {
            for(int t=j+1; t<number.size(); t++) {
                if(number[i] + number[j] + number[t] == 0) answer++; 
            }
                
        
        }
    }
    return answer;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.js
function solution(number) {
    var answer = 0;
    number.sort((a,b) => a-b);
    for(let i=0; i<number.length; i++){
        for(let j=i+1; j<number.length; j++) {
            let k=j+1;
            while(k <= number.length-1) {
                if(number[i]+number[j]+number[k] === 0) answer++;
                k++;
            }
            
        }
    }
    
    return answer;
}